kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/storage/AwsS3StorageInterface.scala

```diff
@@ -34,7 +34,7 @@ import scala.util.Failure
 import scala.util.Success
 import scala.util.Try
 
-class AwsS3StorageInterface(val connectorTaskId: ConnectorTaskId, val s3Client: S3Client)
+class AwsS3StorageInterface(val connectorTaskId: ConnectorTaskId, val s3Client: S3Client, val batchDelete: Boolean)
```

Comment:
writing any unit tests for this class would be appreciated, but totally optional as I realise there aren't currently any.

Reply:
Do you have an example I could copy?